src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../utils/PersonLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  const findPersonByName = (name: string) => {
+    return people.find(person => person.name === name);
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const mother = person.motherName
+            ? findPersonByName(person.motherName)
+            : null;
```

Comment:
Performance issue: This is where the `findPersonByName` function is called for each person. As mentioned before, this can lead to performance issues with large datasets.